src/backend/mpi/swdsm.cpp

```diff
@@ -236,15 +236,28 @@ inline std::size_t align_backwards(std::size_t offset, std::size_t size) {
 	return (offset / size) * size;
 }
 
-void handler(int sig, siginfo_t *si, void *unused){
+void handler(int sig, siginfo_t *si, void *context){
 	UNUSED_PARAM(sig);
-	UNUSED_PARAM(unused);
```

Comment:
does the compiler not complain about the unused context when REG_ERR is not defined?

Reply:
I believe this should now be fixed.